packages/core/src/bus/Event.ts

```diff
@@ -15,3 +15,5 @@ export abstract class Event<T> extends Message<T> {
     return dispatcher.publish<T>(this);
   }
 }
+
+export type EventConstructor<T> = new (...args: any[]) => Event<T>;
```

Comment:
Oh... As discussed, `any` is strictly forbidden
```suggestion
export type EventConstructor<T> = new (...args: unknown[]) => Event<T>;
```